documentation/paths/user.js

```diff
@@ -55,8 +37,23 @@ module.exports = {
                 $ref: '#/components/schemas/Error'
               },
               example: {
```

Comment:
here something more generic, here something more generic, additionally you are already controlling the password with a 422

           `example: {`
               ` message: 'Some params do not have a valid value',`
                `internal_code: 'params_validation_error'`
             ` }`